drive/driveimpl/compositedav/rewriting.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Tailscale Inc & AUTHORS
```

Comment:
This file replaces propfind.go with all rewriting logic, including for PROPFIND and LOCK.